<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <group ns="X4">
    <node pkg="receding_horizon_stitcher" type="receding_horizon_stitcher_node.py" name="path_stitcher" output="screen">
      <param name="rate" value="10.0"/>
      <param name="fixed_frame" type="string" value="world"/>
      <param name="t_horizon" value="4.0" />
      <param name="v_initial" value="1.0" />
      <remap from="odometry" to="odometry"/>
      <remap from="path" to="planned_path"/>
    </node>
  </group>
</launch>
